deploy/static/provider/aws/deploy-tls-termination.yaml

```diff
@@ -38,10 +38,10 @@ metadata:
   namespace: ingress-nginx
 data:
   http-snippet: |
-	server {
-	  listen 2443;
-	  return 308 https://$host$request_uri;
-	}
+      server{
+        listen 2443;
+        return 308 https://$host$request_uri;
+      }
```

Comment:
```suggestion
    server{
      listen 2443;
      return 308 https://$host$request_uri;
    }
```

Reply:
Does this seem ok.
![image](https://user-images.githubusercontent.com/5085914/128054358-716c1467-9312-418c-a03a-85d04bd949a6.png)

Reply:
thnx